express/src/controllers/clients.ts

```diff
@@ -53,7 +53,7 @@ export const showClient: RequestHandler = async (req, res) => {
         token_endpoint_auth_method: client.token_endpoint_auth_method,
         identityVerificationEnabled: identityVerificationEnabled,
         authMethod: client.token_endpoint_auth_method,
-        ...(client.identity_verification_enabled === true && {levelsOfConfidence: client.client_locs.join(" ")}),
+        ...(client.identity_verification_enabled === true && {levelsOfConfidence: client.client_locs ? client.client_locs.join(" ") : []}),
```

Comment:
Does this work?
```suggestion
        ...(client.identity_verification_enabled === true && {levelsOfConfidence: client.client_locs?.join(" ")  ?? []}),
```

Reply:
Yes, it does. I changed default return value [] -> "".